db/schema.rb

```diff
@@ -52,6 +52,21 @@
     t.index ["product_id"], name: "index_categories_products_on_product_id"
   end
 
+  create_table "orders", force: :cascade do |t|
```

Comment:
You might need to do a db:drop, db:create, and db:migrate because this shouldn't be in your schema

Reply:
done!